bodhi/tests/server/services/test_updates.py

```diff
@@ -309,6 +309,20 @@ def test_new_rpm_update_koji_error(self, publish, *args):
         self.assertEquals(up['errors'][0]['description'],
                           "Koji error getting build: bodhi-2.0.0-2.fc17")
 
+    @mock.patch(**mock_valid_requirements)
+    @mock.patch('bodhi.server.notifications.publish')
+    def test_koji_build_config_url(self, publish, *args):
+        """
+        Test html rendering of koji build link using a mock config variable 'koji_build_web_url'
+        """
+        self.app.app.registry.settings['koji_build_web_url'] = u'build_host/param'
+
+        nvr = u'bodhi-2.0.0-2.fc17'
+        resp = self.app.post_json('/updates/', self.get_update(nvr))
+        resp = self.app.get('/updates/%s' % nvr, headers={'Accept': 'text/html'})
+
+        self.assertIn(u'build_host/param', resp)
```

Comment:
Let's assert that the full URL is in there, including the /koji/search and the GET parameters that come after it.

Let's also add one more test to assert that the setting works with or without a trailing slash.